app/lib/tool/test_profile/importer.dart

```diff
@@ -75,10 +75,13 @@ Future<void> importProfile({
   }
 
   // create versions
-  for (final testPackage in profile.packages) {
+  final orderedPackages =
+      _sortByShaHash<TestPackage>(profile.packages, (tp) => tp.name);
+  for (final testPackage in orderedPackages) {
```

Comment:
Why not create them in the order that they have in the test-profile?

Then the order is given when create the test-profile, or the thing that derives the test-profile can derive the order from when the packages were published? So that we have the same ordering as they have on the actual pub site?